ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -2113,3 +2120,102 @@ - (BOOL)isEqual:(id)object {
 }
 
 @end
+
+
+#pragma mark - ORKLocationAnswerFormat
+
+@implementation ORKLocationAnswerFormat
+
+- (instancetype)init {
+    self = [super init];
+    if (self) {
+        _useCurrentLocation = YES;
+    }
+    return self;
+}
+
+- (BOOL)isAnswerValidWithString:(nullable NSString *)text {
+    BOOL isValid = [super isAnswerValidWithString:text];
+    
+    MKPlacemark *placemark = [MKPlacemark ork_placemarkWithString:text];
+    
+    if (placemark && CLLocationCoordinate2DIsValid(placemark.coordinate)) {
```

Comment:
`MKPlacemark` can hold an invalid coordinate, so I believe this is just some defensive programming.  Here's an example:

```objective-c
    CLLocationCoordinate2D location = kCLLocationCoordinate2DInvalid;
    NSDictionary *addressDictionary = nil;
    
    MKPlacemark* mkPlacemark = [[MKPlacemark alloc] initWithCoordinate:location addressDictionary:addressDictionary];

    if (mkPlacemark && CLLocationCoordinate2DIsValid(mkPlacemark.coordinate)) {
        NSLog(@"Valid coordinate");
    }
    else {
        NSLog(@"Invalid coordinate");
    }
```